src/Selenium2Library/__init__.py

```diff
@@ -19,7 +19,10 @@ class Selenium2Library(
     _WaitingKeywords,
     _AlertKeywords
 ):
-    """Selenium2Library is a web testing library for Robot Framework.
+    """Selenium2Library is a web testing library for Robot Framework. This
+	document is about using Selenium2Library. For information about
+	installation, support, and more please visit the
+	[https://github.com/robotframework/Selenium2Library|project page].
 
```

Comment:
Good idea to add this information, but it should be a separate paragraph indented same amount as other paragraphs. For more details see PEP-257: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings